src/main/resources/assets/integrateddynamics/lang/en_us.json

```diff
@@ -1060,6 +1060,8 @@
     "operator.integrateddynamics.string.name.info": "Get the string or name of a named object or converts it to a string",
     "operator.integrateddynamics.string.unique_name": "Unique Name",
     "operator.integrateddynamics.string.unique_name.info": "Get the unique name of an object",
+	"operator.integrateddynamics.string.string_error": "Error",
```

Comment:
Indentation seems to have gone wrong here. Could you use spaces instead of tabs?